playbooks/hosts

```diff
@@ -0,0 +1,2 @@
+[controllers]
+10.190.4.147 ansible_ssh_private_key_file=/root/.ssh/id_rsa_testlab ansible_ssh_extra_args='-o StrictHostKeyChecking=no'
```

Comment:
Remove the real IP address so someone doesn't accidentally clober this machine in the future.